File: algorithm/typescript/BOJ10871.ts
// problem : https://www.acmicpc.net/problem/10871

import fs = require("fs");

const input: string[] = fs.readFileSync("/dev/stdin").toString().split("\n");
const range: number[] = input[0].split(" ").map(Number);
const N: number = range[0];
const X: number = range[1];
const answer: number[] = new Array();

input[1].split(" ").forEach((num) => {
  if (Number(num) < X) {
    answer.push(Number(num));
  }
});

console.log(answer.join(" "));

File: algorithm/python/BOJ10871.py
# problem : https://www.acmicpc.net/problem/10871

import sys

N, X = map(int, sys.stdin.readline().split())
num_list = list(map(int, sys.stdin.readline().split()))

for num in num_list:
    if num < X:
        print(num, end=' ')
